release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -217,6 +217,23 @@ The Memory Manager feature is now enabled by default for all pods running on the
 
 For more information, see xref:../scalability_and_performance/using-topology-manager.adoc#topology_manager_policies_using-topology-manager[Topology Manager policies].
 
+[id="ocp-4-9-scale-latency-tools-for-latency-testing"]
+==== New tools for latency testing
+
+{product-title} {product-version} introduces two tools to measure system latency:
```

Comment:
"... two tools .. ", IIUC , these notes refer ONLY to the new changes that were done in 4.9,am I right?

Reply:
Yes, this is about the 4.9 changes only. The `{product-version}` will become 4.9 when the release is complete.

Reply:
```suggestion
{product-title} {product-version} introduces two additional tools to measure system latency:
```